src/spaceone/dashboard/model/project_dashboard_model.py

```diff
@@ -65,6 +65,10 @@ class ProjectDashboard(MongoModel):
             'labels',
             'project_id',
             'user_id',
-            'domain_id'
+            'domain_id',
+            {
```

Comment:
Setting unique_with as a field property automatically creates a unique index.
So please remove the current index code.